ondewo/nlu/agent.proto

```diff
@@ -387,12 +387,22 @@ message ListAgentsOfUserResponse {
     string next_page_token = 2;
 }
 
+// Defines when to initiate newly created agent when training started
+enum InitiationProtocol {
```

Comment:
Just one thing.

Enum values, by convention, are always upper-case :+1: 
Besides that, it looks good.

Reply:
- [ ]